### PR TITLE
Oleksandr_task#3_print_square_of_digits

### PR DESCRIPTION
**Pull request by Sasha1152**

### CODE REVIEW

**Comment by pivancy** [comment]
Did you try to run your code under Python 2.X and Python 3.X?

**Comment by Sasha1152** [comment]
I tried under Python 3. What a problem?

**Comment by pivancy** [comment]
Please, try to make your code working under both versions.
Thanks!

**Comment by Sasha1152** [comment]
So, now this code works under both version Python, but much slower...

File: Oleksandr_Shmigelskiy/task03_squares_of_numbers.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+start_time = datetime.now()
+
+big_list = range(1000000)
```
Comment (pivancy) [inline Oleksandr_Shmigelskiy/task03_squares_of_numbers.py]: You're using not the test number.

Your solution will work too slow under Python 2.X

**Comment by pivancy** [comment]
> So, now this code works under both version Python, but much slower...

Right! This is the main idea - to make your solution fast enough in both versions. It helps you to understand some basic things.

File: Oleksandr_Shmigelskiy/task03_squares_of_numbers.py (hunk 1 above)
Comment (Sasha1152) [reply Oleksandr_Shmigelskiy/task03_squares_of_numbers.py]: I have remade the code with use xrange function for Python 2

File: Oleksandr_Shmigelskiy/task03_squares_of_numbers.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import sys
+
+from datetime import datetime
+start_time = datetime.now()
+
+big_number = 1000000
```
Comment (pivancy) [inline Oleksandr_Shmigelskiy/task03_squares_of_numbers.py]: Please use the exact number from the task description.

Comment (Sasha1152) [reply Oleksandr_Shmigelskiy/task03_squares_of_numbers.py]: I used that number, but I don't sure whether to start the process of print

**Comment by pivancy** [comment]
👍

**Pull request merged**